prime-router/build.gradle.kts

```diff
@@ -454,7 +455,19 @@ tasks.register("quickPackage") {
     tasks["dokkaHtml"].enabled = false
 }
 
+/**
+ * Docker services needed for running Dockerless
+ */
+dockerCompose {
+    projectName = "prime-router"
+    useComposeFiles.addAll("docker-compose.yml")
```

Comment:
Should also add the file for the postgres server for building.

Reply:
No, not here.  The DB is set to restart automatically, plus it is needed for compiling.  This is mainly for running.